rialto_airflow/funders/linker.py

```diff
@@ -45,8 +45,8 @@ def link_dim_publications(snapshot) -> int:
 
         for row in session.execute(stmt):
             count += 1
-            if count % 100 == 0:
-                logging.info(f"processed {count} publications from Dimensions")
+            if count % 50000 == 0:
+                logging.debug(f"processing publication number {count} from Dimensions")
```

Comment:
as of 1 pm oct 28, this change from `info` to `debug` is uncovered, because the true case for the if is never exercised.